packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -157,6 +157,11 @@ class RunCommand extends RunCommandBase {
         hide: !verboseHelp,
         help: 'Specify the project root directory.',
       )
+      ..addOption(
+        'android-user',
+        hide: !verboseHelp,
+        help: 'If needed specify the android user.',
```

Comment:
Please add much more detail here. "If needed" is never something you need to say when describing command arguments, "android user" isn't very clear, etc. Maybe say something like "The name of the user to pass to "adb"'s "--user" command, which..." (followed by a description of what that command does).